build.gradle

```diff
@@ -51,8 +51,8 @@ allprojects {
 	group = 'org.scenarioo'
     version = getVersionName()
 
-    task wrapper(type: Wrapper) {
-        gradleVersion = '4.7'
+    wrapper {
+        gradleVersion = '4.10.3'
```

Comment:
I do not understand why this is still 4.10 although the PR talked about gradle 5

Reply:
It's a deprecation. I did not upgrade to Gradle 5, simply made it compatible. I could not generate a wrapper anymore, because locally I'm running gradle 5. This does NOT use Gradle 5, just makes it Gradle 5 compatible.

Reply:
Gradle 5 supports Java 8, so we could actually update to Gradle 5. I would prefer if everybody would use the same gradle version, but it's up to you.